aspnetcore/razor-components/index.md

```diff
@@ -20,35 +20,7 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 * Share server-side and client-side app logic all written with .NET.
```

Comment:
I think we should remove the include at the beginning of this doc that adds the note about Blazor being experimental. It's not really relevant for this topic.